From 2680491115af5533418144a67eb8e6c8c5ccab8e Mon Sep 17 00:00:00 2001
From: Deva Ramasubramanian <dramasub@codeaurora.org>
Date: Wed, 8 Jan 2014 14:44:16 -0800
Subject: [PATCH 4/5] mm-video: vidc: Rename *_INDEX_EXTRADATA_* to
 *_EXTRADATA_*

Cleaning up inconsistent naming of extradata enums.

Change-Id: I94ab5fff2e694072d728dc0626cc65358d3beeb4

Backport 2ef926c049b9ef1e950c9d5240e769b62cd4505a to CM11.
Update the two decoder implementations present in CM11; omit absent
software-HEVC and test-app files.
---
 mm-video-v4l2/vidc/vdec/src/omx_vdec_hevc.cpp    | 2 +-
 mm-video-v4l2/vidc/vdec/src/omx_vdec_msm8974.cpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/mm-video-v4l2/vidc/vdec/src/omx_vdec_hevc.cpp b/mm-video-v4l2/vidc/vdec/src/omx_vdec_hevc.cpp
index f4410cf579202297e08d9352e4514ac52bb61f1c..9656d44f0f999e1a0d0b88e0b092ed77aa36abc9 100755
--- a/mm-video-v4l2/vidc/vdec/src/omx_vdec_hevc.cpp
+++ b/mm-video-v4l2/vidc/vdec/src/omx_vdec_hevc.cpp
@@ -7719,7 +7719,7 @@ OMX_ERRORTYPE omx_vdec::enable_extradata(OMX_U32 requested_extradata,
                 DEBUG_PRINT_HIGH("Failed to set panscan extradata\n");
             }
             control.id = V4L2_CID_MPEG_VIDC_VIDEO_EXTRADATA;
-            control.value = V4L2_MPEG_VIDC_INDEX_EXTRADATA_ASPECT_RATIO;
+            control.value = V4L2_MPEG_VIDC_EXTRADATA_ASPECT_RATIO;
             if (ioctl(drv_ctx.video_driver_fd, VIDIOC_S_CTRL, &control)) {
                 DEBUG_PRINT_HIGH("Failed to set panscan extradata\n");
             }
diff --git a/mm-video-v4l2/vidc/vdec/src/omx_vdec_msm8974.cpp b/mm-video-v4l2/vidc/vdec/src/omx_vdec_msm8974.cpp
index 7d86b7b99265af5058d6f770c294f64aff2a4fca..e3c28db3e77859caf33951dd03f83ad9c87f4c6d 100644
--- a/mm-video-v4l2/vidc/vdec/src/omx_vdec_msm8974.cpp
+++ b/mm-video-v4l2/vidc/vdec/src/omx_vdec_msm8974.cpp
@@ -8113,7 +8113,7 @@ OMX_ERRORTYPE omx_vdec::enable_extradata(OMX_U32 requested_extradata,
                 DEBUG_PRINT_HIGH("Failed to set panscan extradata\n");
             }
             control.id = V4L2_CID_MPEG_VIDC_VIDEO_EXTRADATA;
-            control.value = V4L2_MPEG_VIDC_INDEX_EXTRADATA_ASPECT_RATIO;
+            control.value = V4L2_MPEG_VIDC_EXTRADATA_ASPECT_RATIO;
             if (ioctl(drv_ctx.video_driver_fd, VIDIOC_S_CTRL, &control)) {
                 DEBUG_PRINT_HIGH("Failed to set panscan extradata\n");
             }
